api/src/main/java/org/eclipse/microprofile/openapi/annotations/extensions/ExtensionProperty.java

```diff
@@ -0,0 +1,31 @@
+package org.eclipse.microprofile.openapi.annotations.extensions;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * A name/value property within a OpenApi extension
+ *
+ * @see Extension
+ *
+ */
+@Target(ElementType.ANNOTATION_TYPE)
```

Comment:
Can you use MP Config to get hold of the property value?

Reply:
in subsequent PRs we will explore the use of MP Config in a few places.